source/clear-linux/guides/maintenance/autospec.rst

```diff
@@ -1,189 +1,249 @@
 .. _autospec:
 
-Build RPMs with autospec
-########################
+autospec
+########
 
-This guide shows you how to create RPMs with :ref:`autospec <autospec-about>`,
-a tool that assists in automated creation and maintenance of RPM packaging
-on |CL-ATTR|.
+**autospec** is a tool to assist in the automated creation and maintenance of
+RPM packaging in |CL-ATTR|. Where a standard RPM build process using
+:command:`rpmbuild` requires a tarball and :file:`.spec` file to start, autospec
+requires only a tarball and package name to start.
 
-See our :ref:`autospec concept page <autospec-about>` for a detailed explaination
-of how ``autospec`` works on |CL|. For a general understanding of how RPMs work,
-we recommend visiting the `rpm website`_ or the `RPM Packaging Guide`_ .
+.. contents::
+   :local:
+   :depth: 1
 
-Prerequisites
-*************
+Description
+***********
 
-This guide requires that you:
+The autospec tool attempts to infer the requirements of the :file:`.spec` file
+by analyzing the source code and :file:`Makefile` information. It will
+continuously run updated builds based on new information discovered from build
+failures until it has a complete and valid :file:`.spec` file. The autospec tool
+makes use of mock to achieve this. It is possible to influence the exact
+behavior of autospec by providing `control files`_.
 
-* Have installed |CL| on a host machine or virtual environment. For detailed
-  instructions on installing |CL|, visit the :ref:`get-started` section.
+For a general understanding of how RPMs work, visit the `rpm website`_ or the
+`RPM Packaging Guide`_ .
 
-* :ref:`install-tooling`
+How to use
+**********
 
-.. _install-tooling:
+Learn the autospec tool set up and process.
 
-Install the |CL| tooling framework
-==================================
+.. contents::
+   :local:
+   :depth: 1
 
-#. Install the `os-clr-on-clr` developer bundle on your host system.
+Prerequisites
+=============
 
-   .. code-block:: bash
+#. **OS installed**
 
-      sudo swupd bundle-add os-clr-on-clr
+   The |CL| must be installed to use the autospec tool.
 
-#. Download the :file:`user-setup.sh` script.
+#. **Required bundles**
 
-   .. code-block:: bash
+   The autospec tool requires that the :command:`os-clr-on-clr` bundle is
+   installed.
 
-      curl -O https://raw.githubusercontent.com/clearlinux/common/master/user-setup.sh
+Workflow
+========
 
-#. Make :file:`user-setup.sh` executable.
+First time setup
```

Comment:
nit.  Hyphenate "First time"

Reply:
Yes! I appreciate the fresh eyes on these little details - you are always good at these tweeks :) 👍